example/fft_benchmark.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright NumFOCUS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script for benchmarking accelerated ITK FFT performance
+
+import time
+
+import numpy as np
+import itk
+itk.auto_progress(2)
```

Comment:
This is fine as is; there is no need to change it unless you think there is.

However, if this functionality is ever going to be used in a mechanism similar to `pytest` then it might make sense to make sure that none of the code (other than `import` statements) is outside a function -- `pytest` works by invoking a function.  In the present case, this stand-alone script would instead be launched using the typical
```python
if __name__ == "__main__":
    my_top_level_function()
```
mechanism at the end of the file -- which would be compatible with `pytest` because `__name__` would have a different value in that context.

Reply:
I agree this is an easy change that would help with reuse, will update.